lib/testing/src/lib/core/utils/browser-visibility.ts

```diff
@@ -68,4 +68,13 @@ export class BrowserVisibility {
         await this.waitUntilElementIsNotPresent(dialog);
     }
 
+    /*
+     * Wait for specific number of datatable rows by total number of items
+     */
+    static async waitUntilExpectedValue(elementToCheck: ElementFinder, expectedValue: number): Promise<any> {
+        await this.waitUntilElementIsVisible(elementToCheck);
+        const condition = () => elementToCheck.getText().then(value => value && +value.split('of ')[1] < expectedValue);
```

Comment:
if this method is specific for the datatable then I don't think this is the right class and also the name of the method does look is actually explaining is reason to exist

Reply:
I agree it's not the perfect place. I will move it to pagination.